app/serializers/submission_serializer/sections/provider_details.rb

```diff
@@ -16,14 +16,6 @@ def to_builder # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
           end
         end
       end
-
-      private
-
-      def rep_has_partner_declaration
-        return unless crime_application.legal_rep_has_partner_declaration
-
-        crime_application.legal_rep_has_partner_declaration['value']
```

Comment:
@timpeat Can you remember if I had a good argument for having done this? It seems to work for me without now and I've worried myself that I can't find the journey that required it

Reply:
I wonder if it was because the value object attribute was previously not set correctly? Looks like it is now so can go?